tools/data_export/export_remote.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="export_remote" name="Export datasets" version="0.1.0" profile="21.05">
+    <description>to remote files source</description>
+    <!-- TODO
```

Comment:
here is still a TODO is that correct? Do we need to mark this tools as galaxy-env-tool?

Reply:
There's no package yet, but that would allow removing the tool id from https://github.com/galaxyproject/galaxy/pull/11613/files#diff-17974c8048767aa536063088ba97dbc9c4475e7589002c3b2335f9e8490fc920R129